Code/Framework/AzCore/AzCore/std/algorithm.h

```diff
@@ -46,13 +46,18 @@ namespace AZStd
     template<class T>
     constexpr pair<T, T> minmax AZ_PREVENT_MACRO_SUBSTITUTION (const T& left, const T& right) { return AZStd::minmax(left, right, AZStd::less<T>()); }
 
+    // Returns an iterator to the largest element in the range [first, last]
+    using std::max_element;
+    // Returns an iterator to the smallest element in the range [first, last]
+    using std::min_element;
```

Comment:
nit: While we're modifying this could you please also add `minmax_element` (https://en.cppreference.com/w/cpp/algorithm/minmax_element) if it's not here already. Thanks!

Reply:
minmax_element has been implemented

Reply:
We have this implemented in ranges_algorithms.h: [min_element](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/std/ranges/ranges_algorithm.h#L395-L398) and [max_element](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/std/ranges/ranges_algorithm.h#L444-L447)

As @amzn-ahmadkrm mentioned, minmax_element is already implemented in this file

Reply:
Also why was the `std::min_element` and `std::max_element` aliased here.
It is not used in this PR and is unrelated.

In that case this, these changes should be moved to another PR.